src/Altinn.AccessManagement.Integration/Clients/ProfileClient.cs

```diff
@@ -0,0 +1,94 @@
+﻿using System.Net.Http.Headers;
+using System.Security.Cryptography.X509Certificates;
+using Altinn.AccessManagement.Core.Configuration;
+using Altinn.AccessManagement.Core.Constants;
+using Altinn.AccessManagement.Core.Extensions;
+using Altinn.AccessManagement.Core.Helpers;
+using Altinn.AccessManagement.Core.Services.Interfaces;
+using Altinn.AccessManagement.Integration.Configuration;
+using Altinn.AccessManagement.Interfaces;
+using Altinn.Common.AccessTokenClient.Services;
+using Altinn.Platform.Profile.Models;
+using Microsoft.AspNetCore.Http;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Options;
+
+namespace Altinn.AccessManagement.Integration.Clients
```

Comment:
This can be simplified with adding the service for signing provider. Look at events.  I think that is an better copy than app
https://github.com/Altinn/altinn-events/blob/main/src/Events/Services/ProfileService.cs

Reply:
@lovoll  this is relevant for your change